core/modules/system/css/system.theme.css

```diff
@@ -113,8 +113,8 @@ input.form-radio {
   vertical-align: middle;
 }
 .marker,
-.form-required {
-  color: #f00;
+ .form-required {
+   color: #f00;
```

Comment:
Nitpicky, but need to get rid of these extra spaces here on line 116 and 117 before ".form-required" and "color"